### DISCUSSION TITLE
Add widget button not showing

### DISCUSSION THREAD

**Discussion by tamer-yigit** [OrchardCMS/OrchardCore#12581] [Q&A] [answered]
I try to make a web site with Orchard Cms. I want to add a widget for a page (like our customer testimonial slide with stars etc you know) But I could not see add widget button on Edit Widget and Layers screen. Why? I selected Blank recipe when install screen. Could it be related this? How I can activated it.

Thanks advance

![image](https://user-images.githubusercontent.com/6212666/194503966-8a4bc0ca-e175-4eb8-b408-133aa14ebf54.png)

**Comment by lampersky** [comment, marked as answer]
@tyigit you need to create **ContentType** with stereotype **Widget** first

**Comment by tamer-yigit** [comment]
@lampersky Thank you so much

**Reply by Piedone** [reply]
Next time, please open discussions for questions in the Q&A category, so answers can be marked as answers: https://github.com/OrchardCMS/OrchardCore/discussions/categories/q-a

I moved it to there now; please accept Dawid's answer :).
